encoders/nlp/TextPaddlehubEncoder/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina:devel
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN apt-get update && \
+    apt-get -y install libgomp1 libgl1-mesa-glx libglib2.0-0
+
+# install the third-party requirements
+RUN pip install -r requirements.txt && pip uninstall -y pathlib
+
+# for testing the image
+RUN pip install -y pytest pytest-mock mock && JINA_TEST_PRETRAINED='true' pytest
```

Comment:
```&& EXPORT JINA_TEST_PRETRAINED='true'``` instead of ```JINA_TEST_PRETRAINED='true'```

Reply:
it actually seems to work without export

Reply:
- `-y` is necessary for uninstallation.
- `EXPORT` is not necessary